.buckversion

```diff
@@ -0,0 +1 @@
+6893d2f82e
```

Comment:
What happens if you don't have a `.buckversion` file ? Adding this file like this will conflict with our internal setting.